userspace/libscap/ringbuffer/ringbuffer.h

```diff
@@ -23,6 +23,13 @@ limitations under the License.
 #include "barrier.h"
 #include "sleep.h"
 
+static unsigned long per_cpu_buffer_dim;
```

Comment:
A `static` variable in a header file? Shouldn't this be `extern` and the variable be defined in some `.c` file? Otherwise you will get a copy of the variable for each source that includes this.

Reply:
Agree with Mauro, even if this is not an issue since the ringbuffer api will be used by just one engine at runtime.

Reply:
yeah agree it's better, thank you :)

Reply:
Done :)